docs/actions.md

```diff
@@ -340,7 +344,7 @@ is skipped if an action is dispatched to a previously initialized container ---
 You can tell if an [invocation was a warm activation or a cold one requiring initialization](annotations.md#annotations-specific-to-activations)
 by inspecting the activation record.
 - An action runs for a bounded amount of time. This limit can be configured per action, and applies to both the
-initialization and the execution separately.
+initialization and the execution separately. If the action time limit is exceeded the activation's response status is _application error_.
```

Comment:
This changes in #3941 - @tysonnorris @markusthoemmes please confirm.

Reply:
I think it only changes for the init phase.

Reply:
I suggest:

If the action time limit is exceeded during initialization, the activation's response status is ... otherwise it is ...